packages/arbitrage-bot-cli/src/cli.ts

```diff
@@ -8,6 +8,25 @@ import yargs from 'yargs';
 import { hideBin } from 'yargs/helpers';
 import { asciLogo } from './asciLogo';
 import { exportConfigToFile, loadConfigFromFile } from './configToJson';
+const configPath = '/config.json';
+
+const list = async () => {
+  const configFromFile = loadConfigFromFile(process.cwd() + configPath);
```

Comment:
Please extract path to config file to top of this file. It is reused in `start` command as well